pkgs/by-name/ge/get-google-fonts/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, fetchpatch
+}:
+
+buildNpmPackage {
+  pname = "get-google-fonts";
+  version = "unstable-2020-06-30";
```

```suggestion
  version = "1.2.2-unstable-2020-06-30";
```

https://github.com/MrMaxie/get-google-fonts/commit/8275ff84fdc1d376d764b61542b52b63e4981e79